src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -33,7 +32,7 @@ import {getChannelListInfo} from '../../utils/string/StringUtils';
 
 const useCoreChatSystemHook = () => {
   const {localDb, refresh} = useLocalDatabaseHook() as UseLocalDatabaseHook;
-  const {anonProfileId, signedProfileId, profile} = useProfileHook();
+  const {anonProfileId, signedProfileId} = useProfileHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `profile` from the destructuring assignment of `useProfileHook()` might cause issues if `profile` was used elsewhere in the code. Please ensure that `profile` is not used anywhere else in this file or consider adding it back if it's needed.
```diff
-  const {anonProfileId, signedProfileId} = useProfileHook();
+  const {anonProfileId, signedProfileId, profile} = useProfileHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->